src/api/vdom.js

```diff
@@ -89,7 +89,7 @@ const attributesContext = propContext(attributes, {
 
   // Default attribute applicator.
   [symbols.default] (elem, name, value) {
-    const { props, prototype } = customElements.get(elem.tagName) || {
+    const { props, prototype } = customElements.get(elem.localName) || {
```

Comment:
It's case-sensitive, but `localName` should return the exact name of the element how it was specified in HTML, as I understand. More info here https://github.com/w3c/webcomponents/issues/604.